Content.Server/Traits/Assorted/UnborgableComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Server.Traits.Assorted;
+
+/// <summary>
+/// This is used for the unborgable trait.
+/// </summary>
+[RegisterComponent]
+public sealed partial class UnborgableComponent : Component
+{
+
+}
```

Comment:
doesn't need a body

Reply:
I literally just copy pasted the unclonable component and changed it to unborgable. They all look like this.

Reply:
still doesnt need a body, you can just use a semicolon instead